pkg/resources/strings.go

```diff
@@ -64,3 +64,13 @@ func SafeStringDereference(s *string) string {
 	}
 	return ""
 }
+
+// String returns a pointer to the string value passed in.
+func String(v string) *string {
+	return &v
+}
+
+// Int32 returns a pointer to the int32 value passed in.
+func Int32(v int32) *int32 {
+	return &v
+}
```

Comment:
It could be worth using this k8s library here: https://pkg.go.dev/k8s.io/utils/pointer
I think I used it in integreatly-operator. It's worth agreeing on a standard way of doing this type of operation as each project may have its own implementation of these functions, which could result in code pollution.

Reply:
Agreed, good shout I'll make that change